config/settings/base.py

```diff
@@ -30,15 +30,25 @@
     'django.contrib.messages',
     'django.contrib.staticfiles',
     'django.contrib.humanize',
+    'django.contrib.sites',
 
     # local apps
     'common',
     'comments',
     'recipes',
     'api',
+    'social_login',
```

Comment:
I don't think we need the app at all, it's got nothing in it and we shouldn't need a new app to do login since everything we need is included in the `allauth` app